docs/src/docs/versus.md

```diff
@@ -0,0 +1,80 @@
+# How KitOps is Different
+
+When people first come across KitOps they sometimes wonder, "how is this better than my favorite MLOps tool, git, Docker, etc...?"
+
+## How AI Project Assets are Managed Today
+
+Most teams working on AI projects store, track, and version their assets in one of two ways.
+
+1. Using an MLOps tool
+1. Using a combination of git, containers, block storage, and Jupyter notebooks
```

Comment:
Is this supposed to be "2."?

Reply:
No, one of my favourite things about markdown is that you can do a numbered list with all 1. at the start and when it's processed it'll automatically give them the right number in sequence. That way you can move numbered points around anytime and know you'll always end up with a nice sequential list.